api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -115,13 +115,49 @@ private void sendWithRetries(MandrillMessage msg, String description) throws Mes
         default:
           if (retries == 0) {
             log.log(Level.SEVERE, String.format(
-                "Email '%s' was not sent. Default case.", description));
+              "Email '%s' was not sent. Default case.", description));
             throw new MessagingException("Sending email failed: " + attempt.getRight().toString());
           }
       }
     } while (retries > 0);
   }
 
+  private MandrillMessage buildWelcomeMessage(String contactEmail, String password, User user) throws MessagingException{
+    MandrillMessage msg = new MandrillMessage();
+    RecipientAddress toAddress = new RecipientAddress();
+    toAddress.setEmail(contactEmail);
+    msg.setTo(Collections.singletonList(toAddress));
+    try {
+      String msgHtml = buildEmailHtml(password, user);
+      msg.setHtml(msgHtml);
+      msg.setSubject("Your new All of Us Account");
+      msg.setFromEmail(workbenchConfigProvider.get().mandrill.fromEmail);
+      return msg;
+    } catch (IOException e) {
+      throw new MessagingException("Error reading in email");
+    }
+  }
+
+
+  private String buildEmailHtml(String password, User user) throws IOException {
+    CloudStorageService cloudStorageService = cloudStorageServiceProvider.get();
+    StringBuilder contentBuilder = new StringBuilder();
+    URL emailContent = Resources.getResource(WELCOME_RESOURCE);
+    Resources
+      .readLines(emailContent, StandardCharsets.UTF_8)
+      .forEach(s -> contentBuilder.append(s).append("\n"));
+    String string = contentBuilder.toString();
+    Map<String, String> replaceMap = new HashMap<>();
```

Comment:
opt_nit: I still think the map builder would be better, not sure whether that comment was missed or just not accepted